recipe/meta.yaml

```diff
@@ -13,13 +13,11 @@ source:
   sha1: afde507fbee85a1aeb5fcb32e897bd3ea2bc48f5  # [win]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - python
-    - python.app  # [osx]
```

Comment:
@astrofrog can you take a look at this? Would it be acceptable without `pythonw` and `python.app`? Are there alternatives without the `python.app` package?